lib/block-supports/layout.php

```diff
@@ -63,10 +63,16 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 
 	$style = '';
 	if ( $content_size || $wide_size ) {
-		$style  = ".wp-container-$id > * {";
+
+		$style  = ".wp-container-$id {";
+		$style .= 'display: grid;';
+		$style .= '}';
```

Comment:
Nitpick: Maybe this would be easier to read as a single liner?
```suggestion
		$style  = ".wp-container-$id { display: grid; }";
```